.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Thank you for contributing to The Foreman, please read the following guide
+available at https://www.theforeman.org/contribute.html - in short:
+
+* Create an issue at https://projects.theforeman.org/projects/foreman/issues
+* Reference the issue via "Fixes #1234" or "Refs" in the commit message
+* Extract all strings for i18n, see Translation section in the guide
+* Prepend `[WIP]` to prevent lengthy tests for work in progress
```

Comment:
The tests still run on ```[WIP]```. It just doesn't check the commit message style. This allows you to add commits while working on it and squash it when you think it's ready.